string dataset_path
---
bool success
string message
